MSON Specification.md

```diff
@@ -794,6 +795,32 @@ by using an `optional` attribute and/or MAY indicate values are samples using a
     Implies an `array` type structure that MUST contain "red" as an item and MAY contain any other strings, where
     "green" is a sample value.
 
+With a `fixed-type` _[Type Attribute][]_:
+
+- If a _[Named Type][]_ or _[Member Type][]_ annotates its type as `fixed-type`, _[Nested Member Types][]_ do not inherit
+the `fixed-type` attribute.
```

Comment:
Okay, I have looked over this and I agree with `fixed-type` not being inheritable.